metricbeat/module/mongodb/_meta/replicaset.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+echo "Waiting for startup.."
+until curl http://mongodb:27017/serverStatus\?text\=1 2>&1 | grep uptime | head -1; do
+  printf '.'
+  sleep 1
+done
+
+echo curl http://mongodb:27017/serverStatus\?text\=1 | grep uptime | head -1
+echo "MongoDB Started.."
+
+mongo --host mongodb:27017 --eval "if(rs.status().code == 94) rs.initiate();"
```

Comment:
Could `rs.initiate()` be executed by the integration test instead of needing a new image just for this command?

Reply:
I didn't want to risk by changing the entry point of the current mongodb service.
We can initiate replica set from the integration test. But I wasn't sure that this is a good idea or not.

Reply:
Could you try the option of running `rs.initiate()` in the integration test? If it doesn't affect other things I think I prefer that than having an additional image and service.